modules/httpcache/httpcache.go

```diff
@@ -17,16 +17,26 @@ import (
 )
 
 // AddCacheControlToHeader adds suitable cache-control headers to response
-func AddCacheControlToHeader(h http.Header, d time.Duration) {
+func AddCacheControlToHeader(h http.Header, d time.Duration, noTranform bool) {
+	directives := []string{}
```

Comment:
How about using `AddCacheControlToHeader(h http.Header, d time.Duration, directives ...string)`?

Then calling `AddCacheControlToHeader(..., "no-transform")` seems easier to read than `AddCacheControlToHeader(..., true/fase)`  and it's easier to add more customized options by caller in the feature.

Otherwise L-G-T-M

Reply:
Will do